spec/workers/emotion_selection_notification_worker_spec.rb

```diff
@@ -12,16 +12,24 @@
       expect(worker.users).to match_array([user1, user2])
     end
     it 'fetch time_period' do
-      expect(worker.time_period).to eq(time_period)
+      expect(worker.time_period).to eq(TimePeriod.current)
     end
   end
 
   describe '.run_notifications' do
     it 'sends email notifications' do
+      ENV["DAY_TO_SEND_INVITES"] = Date.current.strftime("%A")
```

Comment:
I don't think you can set ENV variables like this in tests (unfortunately). Here are a few workarounds I've done in the past:
 * Use gem [climate_control](https://github.com/thoughtbot/climate_control)
 * Have a private method in EmotionSelectionNotificationWorker return the ENV variable, and then use mocks for that method, e.g. `allow(EmotionSelectionNotificationWorker).to receive(:day_to_send_invite).and_return(Date.current.strftime("%A").downcase)`
 * You can mock the ENV variable itself (but I've found that sometimes this is flaky): `allow(ENV).to receive(:[]).with('DAY_TO_SEND_INVITES').and_return(Date.current.strftime("%A")`